recipe/entry_point.py

```diff
@@ -56,7 +63,15 @@ def map(self, func, *iterables):
            import tqdm
            from conda_package_handling import api
            from concurrent.futures import ProcessPoolExecutor
-           executor = ProcessPoolExecutor()
+
+           if args.num_processors:
+               if args.num_processors < 1:
```

Comment:
This condition will never be true due to the first check. Usually `0` means "all processors", though? So maybe it's better yo handle that use case. If not, argument validation should happen as part of the `argparse` logic with an `Action` or similar.

Reply:
@jaimergp : I refactored the validation to use the argparse machinery and cleaned up the logic a bit. In the case of `ProcessPoolExecutor`, passing a 0 will raise an error; `max_workers` must be greater than/equal to 1.